opentracing_shim/src/main/java/io/opentelemetry/opentracingshim/TraceShim.java

```diff
@@ -34,7 +34,7 @@ private TraceShim() {}
   public static io.opentracing.Tracer createTracerShim() {
     return new TracerShim(
         new TelemetryInfo(
-            OpenTelemetry.getTracerFactory().get("openTracingShim"),
+            OpenTelemetry.getTracerFactory().get("opentracingshim"),
```

Comment:
any particular reason to change this to all lowercase?

Reply:
I need to be consistent between here and the test. Happy to go with any of the ways  but need to be the same to make test pass.